### PR TITLE
[Refactor] Big refactor

### PR DESCRIPTION
**Pull request by Swiizyy**
### Refactors
- Added Scheduled Jobs (Role removal, BirthdaysChecker)
- Renamed `container.sequelize` to `container.db`
- Created a BirthdayyBot class & abstracted options
 
Closes: #9
Closes: #28

### ToDos:
- [x] config
- [ ] sequelize model
- [ ] birthdayEvents switch to listener
- [ ] registerCommands
- [ ] routesOptions
- [ ] routes

### CODE REVIEW

File: src/lib/BirthdayyBot.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { container, SapphireClient } from '@sapphire/framework';
+import { Sequelize } from 'sequelize';
+import { BD_OPTIONS, CLIENT_OPTIONS } from '../config';
+
+export class BirthdayyBot extends SapphireClient {
+    public constructor() {
+        super(CLIENT_OPTIONS);
+
+        container.db = new Sequelize(BD_OPTIONS);
```
Comment (nikolaischunk) [inline src/lib/BirthdayyBot.ts:9]: add a log that the connection to the db was established

**Comment by nikolaischunk** [comment]
Please split those tasks in small PRs that we  can merge one by one and avoid large mergeconflicts!

**Comment by Swiizyy** [comment]
So I delete this one and redo it in fragment?

**Comment by nikolaischunk** [comment]
> So I delete this one and redo it in fragment?

Yes, makes more sense :)

**Pull request closed without merging**